src/wechaty/wechaty.py

```diff
@@ -693,7 +694,7 @@ async def scan_listener(payload: EventScanPayload) -> None:
                         log.info(
                             'or you can scan qrcode from: '
                             'https://wechaty.js.org/qrcode/%s',
-                            qr_code
+                            quote(qr_code)
```

Comment:
We should keep the qr code data clean at here.

The url encode should not be put at here because its only related to the web url.

You can put the url encode to the dong-dong bot example.